src/breeze/components/breeze/navbar_links.cr

```diff
@@ -0,0 +1,8 @@
+class Breeze::NavbarLinks < BreezeComponent
```

Comment:
Renamed from SidebarLinks because the links show up at the top of the page, and not the side 😂